scss/main.scss

```diff
@@ -156,4 +163,69 @@
 
 #main-call-to-action {
   margin: 15px auto 0 auto;
+}
+
+.fb-icon, .insta-icon {
```

Comment:
give the icons in the navbar the call to action styles so that they will stand out

Reply:
the box-shadow does not look good when this is done cuz the images are not perfect squares but their containers are, I can make them black on hover over, though then it kind of seems off since none of the other items get black fill/color on hover